SR2023/2022-09-29-confirm-team-details.md

```diff
@@ -0,0 +1,24 @@
+---
+to: SR2023 sign-ups
+subject: Confirming your details for SR2023
+---
+
+Hello!
+
+Thank you for your interest in Student Robotics 2023. We would like to offer you a place. In order to do that, we have a few questions and extra pieces of information we need.
+
+1. Can you confirm you are over the age of 18, and officially related to the institution you are bringing a team from (eg teacher at a school, leader at a scout group etc).
+2. A contact phone number. We will only contact you by phone for urgent things, please ensure it is a number that you would have with you at events
+3. Could you provide a second set of contact details for someone at your institution (eg head of department). They should know about the competition, and can be used as a backup contact.
+   1. Name
+   2. Email address
+   3. Contact phone number
+4. Are you able to attend a physical competition in the UK on a weekend in early April?
```

Comment:
Once we're updated them so the content is all in, it's probably worth reviewing these so that they're all framed the same way. Specifically: currently some are questions, others aren't.